lighthouse-core/test/report/html/renderer/psi-test.js

```diff
@@ -66,7 +66,7 @@ describe('DOM', () => {
         const roundTripLHResult = /** @type {LH.Result} */ JSON.parse(sampleResultsRoundtripStr);
         const result = prepareLabData(roundTripLHResult, document);
 
-        // sanity check that the report exists and has some content
+        // Check that the report exists and has some content.
```

Comment:
I could add "Soundness" or something before all these if someone feels like it adds to the comment.